exercises/grep/description.md

```diff
@@ -1,77 +1,25 @@
-# Description
+# Instructions
 
-Search a file for lines matching a regular expression pattern. Return the line
-number and contents of each matching line.
+Search files for lines matching a search string and return all matching lines.
 
-The Unix [`grep`](http://pubs.opengroup.org/onlinepubs/9699919799/utilities/grep.html) command can be used to search for lines in one or more files
-that match a user-provided search query (known as the *pattern*).
+The Unix [`grep`](http://pubs.opengroup.org/onlinepubs/9699919799/utilities/grep.html) command searches for lines in files that match a regular expression. Your task is to implement a simplified `grep` function, which supports searching for literal strings.
 
-The `grep` command takes three arguments:
+The `grep` function takes three arguments:
```

Comment:
It's a command. That in rust (perhaps?) you implement an equally named _function_ is an implementation detail. In various tracks, like JavaScript, you write a working program that has a `grep` command (not a function).

Reply:
Done.